appendix_meta_rules.md

```diff
@@ -605,16 +620,90 @@ See Detection in [sigma specification](Sigma_specification.md)
 Example
 
 ```yaml
-title: Filter Administrator account 
-description: The valid administrator account start with adm_ 
-action: filter
-type: exclude
-rules: 
-    - 6f3e2987-db24-4c78-a860-b4f4095a7095 # Data Compressed - rar.exe
-    - df0841c0-9846-4e9f-ad8a-7df91571771b # Login on jump host
+title: Filter Administrator account
+description: The valid administrator account start with adm_
 logsource:
     category: process_creation
-    product: windows 
-selection:
-    User|startswith: 'adm_'
+    product: windows
+global_filter:
+  rules:
+    - 6f3e2987-db24-4c78-a860-b4f4095a7095 # Data Compressed - rar.exe
+    - df0841c0-9846-4e9f-ad8a-7df91571771b # Login on jump host
+  selection:
+      User|startswith: 'adm_'
+  condition: selection
 ```
+
+# Examples
+This section gives complete examples in order to make it easier for people new to Sigma to get started and for showcasing new features of the Sigma standard. Use them as a blueprint for your own ideas.
+
+## Correlation
+
+
+### Failed Logins Followed by Successful Login
+
+The following Correlation describes a use case in which an attacker successfully performs a brute-force attack. This example helps in showcasing some highlights:
+ - You can use YAMLs multi document feature (`---`) to have everything grouped together in one file
+ - Rules can be referenced in a human-friendly way using their unique `name`.
+ - Correlations can be used recursively to express more complex use cases
```

Comment:
```suggestion
 - Correlations can be chained to express more complex use cases
```

Reply:
I prefer the term "chained" in favor of "recursively" because recursion means usually something that can also be done in a self-referencing and also endless way.